tests/ImageSharp.Tests/TestUtilities/ImageComparison/TolerantImageComparer.cs

```diff
@@ -105,12 +106,12 @@ public TolerantImageComparer(float imageThreshold, int perPixelManhattanThreshol
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static int GetManhattanDistanceInRgbaSpace(ref Rgba32 a, ref Rgba32 b)
+        private static int GetManhattanDistanceInRgbaSpace(ref Rgba64 a, ref Rgba64 b)
```

Comment:
👍 seems good, but it might be worth to have another look tomorrow. All our tests depend on this stuff 😄

Reply:
Defo would like a second opinion on this. I had to change some tolerances, I'm assuming code round actually hid differences before.